Assets/Scripts/Tank/TankController.cs

```diff
@@ -7,19 +7,36 @@ namespace TankBattle.Tank
 {
     public class TankController
     {
+        private bool isBotTank = false;
+        private bool isLoaded;
+        public bool IsBotTank {get {return isBotTank; }}
+        private TankBotState currentBotState;
         private TankModel tankModel;
         private TankView tankView;
         public TankController(TankScriptableObject _tankScriptableObject, Vector3 position)
         {
             tankModel = new TankModel(_tankScriptableObject);
             tankView = GameObject.Instantiate<TankView>(tankModel.TankPrefab, position, Quaternion.identity);
             tankView.SetController(this);
+            isLoaded = true;
```

Comment:
what is this isLoaded for? please explain offline once

Reply:
I was just trying to introduce a reload time for the tank. So the tank will be fired only after reloading.